inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/listener/sort/StreamSortConfigListener.java

```diff
@@ -62,18 +60,15 @@ public TaskEvent event() {
     @Override
     public boolean accept(WorkflowContext context) {
         ProcessForm processForm = context.getProcessForm();
+        String className = processForm.getClass().getSimpleName();
         String groupId = processForm.getInlongGroupId();
-        if (!(processForm instanceof GroupResourceProcessForm)) {
-            LOGGER.warn("zookeeper enabled was [false] for groupId [{}]", groupId);
+        if (processForm instanceof GroupResourceProcessForm || processForm instanceof StreamResourceProcessForm) {
```

Comment:
Only need to check `StreamResourceProcessForm`, because the following `listen` method casts `ProcessForm` to `StreamResourceProcessForm`.